Sources/PexipRTC/Internal/Capturers/WebRTCScreenCapturer.swift

```diff
@@ -80,17 +85,22 @@ final class WebRTCScreenCapturer: RTCVideoCapturer, ScreenMediaCapturerDelegate
         _ capturer: ScreenMediaCapturer,
         didCaptureVideoFrame videoFrame: VideoFrame
     ) {
-        let cropDimensions = videoFrame.contentDimensions
+        #if os(iOS)
+
+        // Screen frames are always portrait while video profile is landscape.
```

Comment:
Is it also the case when the device is in landscape?

Reply:
Yes, I tested in both portrait and landscape, and it looks like video frames we get from broadcast upload extension are always "portrait". Then we can use `CGImagePropertyOrientation` from `CMSampleBuffer` to get the actual orientation.